tracer/test/Datadog.Trace.Tests/LambdaCommonTests.cs

```diff
@@ -25,36 +27,44 @@ public class LambdaCommonTests
         public async Task TestCreatePlaceholderScopeSuccessWithTraceIdOnly()
         {
             await using var tracer = TracerHelper.CreateWithFakeAgent();
-            var scope = LambdaCommon.CreatePlaceholderScope(tracer, "1234", null);
 
+            var myHeaders = new Dictionary<string, string> { { HttpHeaderNames.TraceId, "1234" }, };
+            var scope = LambdaCommon.CreatePlaceholderScope(tracer, myHeaders);
             scope.Should().NotBeNull();
             scope.Span.TraceId128.Should().Be((TraceId)1234);
             ((ISpan)scope.Span).TraceId.Should().Be(1234);
             scope.Span.SpanId.Should().BeGreaterThan(0);
         }
 
         [Fact]
-        public async Task TestCreatePlaceholderScopeSuccessWithSamplingPriorityOnly()
```

Comment:
That's right. A trace id is required when extracting trace context from headers, otherwise we ignore all the other data. But what changed here? Are we now trying to propagate trace context without a trace id?

Reply:
> But what changed here?

Oh, I see. The `null` in `LambdaCommon.CreatePlaceholderScope(tracer, null, "-1")` below was the trace id, and it was handled differently since the previous code was not using `SpanContextPropagator`.